.changeset/four-snails-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': minor
+---
+
+Added support for federated credentials using managed identities in the Azure DevOps integration.
```

Comment:
Do we want to note this is Cloud only or is that pretty obvious for Azure DevOps Server users?

Reply:
This does not necessarily apply to Azure DevOps Cloud, only to Azure DevOps Cloud backed by Entra ID. I'll mention Entra ID here